README.md

```diff
@@ -1,3 +1,46 @@
 # Get-Originals Rewriter Codemod
 
 [![Build Status](https://travis-ci.com/jackbsteinberg/get-originals-rewriter.svg?branch=master)](https://travis-ci.com/jackbsteinberg/get-originals-rewriter)
+
+## Usage
+
+To use the Get-Originals Rewriter, run:
+
+```
+npm run rewrite PATH
+```
+
+where `PATH` is a file or directory of files to be rewritten.
+
+The Get-Originals Rewriter provides the following options:
+
+```
+Usage: npm run rewrite PATH...
+  or:  npm run rewrite -- [OPTION]... PATH...
+  or:  npm run rewrite -- [OPTION]... --stdin < file_list.txt
+
+Applies rewriter logic recursively to every PATH.
+If --stdin is set, each line of the standard input is used as a path.
+
+Options:
```

Comment:
Is this just the output of --help? If so, I don't think it's a good idea to include it in the README as it will need updating every time it changes. Better to just explain how get it to dump out the current set of options.

Poking a bit more, it seems like doing `npm run rewrite --help` gets you nothing but adding script entry of

```
  "rewrite-help": "jscodeshift -h"
```

works, so maybe point people at that.

Reply:
How do you anticipate it changing?

Per the README you need -- OPTIONS, e.g. -- --help.

Reply:
Ah `npm run rewrite -- --help` does the right thing.

> How do you anticipate it changing?

How do you anticipate it never changing? Unless I'm missing a reason why this is all set in stone, it seems highly unlikely that the flags and their help text will never change. All of that content (flag names, values, help-text) is already checked in, the principle of "don't repeat yourself" says we we shouldn't check it in again in a new location.

Reply:
It would change if we bump the jscodeshift version, which seems rare (very few new versions recently).

I think there's generally a tradeoff between "don't repeat yourself" and "have good documentation"; after all documentation is often just a repetition of what you can glean from the source code. My personal opinion is that any CLI tool on GitHub should state how you use and configure it in the README, instead of requiring you to clone the project, install its dependencies, and then run its `--help` to find that information.